Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -200,6 +201,30 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcSucceeds()
         }
 
         [TestMethod]
+        public void Exe_PostProc_ExecutableNotFound_PostProcFails()
+        {
+            // Arrange
+            string rootDir = TestUtils.CreateTestSpecificFolder(this.TestContext);
+            using (InitializeNonTeamBuildEnvironment(rootDir))
+            {
+                string binDir = CalculateBinDir(rootDir);
+
+                MockBuildAgentUpdater mockUpdater = CreateValidUpdater(binDir, "http://anotherHost");
+                
+                // Act
+                TestLogger logger = CheckExecutionFails(mockUpdater, "end");
+
+                IBootstrapperSettings settings;
+                ArgumentProcessor.TryProcessArgs(new string[] { "end" }, logger, out settings);
+
+                // Assert
+                mockUpdater.AssertUpdateNotAttempted();
+                mockUpdater.AssertVersionNotChecked();
+                logger.AssertErrorLogged(String.Format(Resources.ERROR_PostProcessExeNotFound, settings.PostProcessorFilePath));
```

Comment:
It's sufficient to check that errors were logged, so you don't need the settings object